sellercenter-seller/src/main/java/com/gfg/sellercenter/seller/Seller.java

```diff
@@ -0,0 +1,29 @@
+package com.gfg.sellercenter.seller;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.NotNull;
+import java.io.Serializable;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@EqualsAndHashCode
+public class Seller implements Serializable {
+    @NotNull
+    private Integer id;
+
+    private String email;
```

Comment:
I think this should be `NotNull`

Reply:
User's email can't be null, seller email can be null